public/api/conf/1.0/examples/get-user-info-example-1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "given_name" : "Thomas",
+  "family_name" : "Delgado",
+  "address" : {
+    "formatted" : "221B Baker Street\nLondon\nNW1 9NT\nGreat Britain",
+    "postal_code" : "NW1 9NT",
+    "country" : "Great Britain"
+  },
+  "birthdate" : "1985-04-22",
+  "uk_gov_nino" : "WE456325C"
```

Comment:
They are both from the existing docs - i.e. randomly generated. So they *could be* real.
I'll wait for Tony's real fake NINO and use that.